### ISSUE TITLE
Läs på om Brewmeister

### ISSUE THREAD

**Issue by sweco-seukni** [sweco-seukni/Bbbolaget#2]
Ta reda på mer om Brewmeister, om man hittar en länk eller liknanade med intressant information så är det nog bäst att skapa en ny ticket Brewmeister och skriva om det där

**Comment by Bbbolaget-karl** [comment]
ticket?

**Comment by Bbbolaget-karl** [comment]
finns en liten film på tuben, här, 
https://www.youtube.com/watch?v=UlLcwvbYLiE
finns flera. 

Tydligen lakar man inte, det är inräknat! Enklare än förväntat

**Comment by Bbbolaget-karl** [comment]
en rätt lång film, men man vaggas in i tempot. Är väl som att brygga tänker jag
https://www.youtube.com/watch?v=KqA4d605XcQ

**Comment by Bbbolaget-karl** [comment]
Jo. kollat lite förutsättningar för det man behöver om man handlar en sån här. 
Ev kommer vi behöva en ny kylspiral då vår antagligen inte når ner så långt som behövs. finns för 1100 extra. eller bygga en ny. Alternativt köper man en plattvärmeväxlare (plattis) och tar kylningen på utflödet. Smidigast egentligen. Samma pengar ungefär. Får utvärdera befintligt först (om vi kommer dit)

men summa på måsteprodukter, som jag ser det är
*braumeister 15900
*humlesil 795
*huva 1490
summa 18185

Därutöver måste jag koppla huvan ut nånstans så småningom. men där är vi inte än. De kostnaderna kan kanske tillfalla huset. Får återkomma om det. Kan också bli billit.

**Comment by sweco-seukni** [comment]
Fint, vi borde kunaa modda vår befintliga kykspiral så att den går längre ner, hade varit bra om den funkar till nuvarande gryta också isf.

Finns det några mer kanskeprodukter elker löpande kostnader som kan tillkomma?

**Comment by Bbbolaget-karl** [comment]
Jag hoppas också vi kan modda den befintliga kylröret. 
Man kan om man vill köpa en snygg mantel till BMen, så slipper man isolera med liggunderlag. Men vi kan fortsätta på vårt vinnande koncept där. lika funktionellt. 

Lindh har bilder för sin BM 50 och utrustning. Fina bilder som vanligt. inspiration och andra nyttoföremål?
http://www.lindhcraftbeer.com/min-utrustning/

Det är väl möjligen rengöringskitet för 595 extra. Inget speciellt egentligen förutom ett par borstar som är anpassade för ändamålet. Ibland tänker jag det är dumt att snubbla på målsnöret och beställer allt möjligt, ibland är det onödigt. Säger ni andra?

tror inte det är något mer.

**Comment by Bbbolaget-karl** [comment]
det finns de som byter tappkran också. 795 extra då. men det där är överkurs. märker vi om ett år eller så om det är nödvändigt.
